### ISSUE TITLE
Fix issue with scorecard "customize indicators" feature

### ISSUE THREAD

**Issue by james-minton** [Hyperobjekt/spi-map#56]
The "customize indicators" feature in the scorecard behaves differently than it does on the map, in a buggy way.

_Just to be clear: the feature on the map view behaves correctly; the incorrect behavior is on the **scorecard** view._

Undesired behavior: Checking the box for any of the higher level categories (such as Nutrition & Basic Medical Care) will _exclude_ it from the scorecard. But checking the box for _subsidiary_ categories (e.g. "households with food stamp benefits") will limit that the scorecard indicators to just that one—which is correct.

Desired behavior: Checking a box for a top level indicator(s) will filter the scorecard to show just that indicator (not exclude it). Refer to how the same selection behaves in the map interface.

<img width="455" alt="Screen Shot 2023-06-22 at 7 47 32 PM" src="https://github.com/Hyperobjekt/spi-map/assets/33635008/bc9b84a8-b00e-4bfc-bde1-09b8462a32c0">

**Comment by erbnando** [comment]
Hi @james-minton,

I assumed these changes were to `development` so I pushed to the branch I branched off `development`. Let me know if they're supposed to be off a different branch. As far as the update, it was one of those times when a one-line fix takes hours. There was one function export that was misnamed -- https://github.com/Hyperobjekt/spi-map/pull/53/files#diff-16a52095a9a93357c346f61dcdde84c542f5e92977a1f30a4adf45fc6b834872L57 -- that made it difficult to follow the breadcrumb trail to actual fix at first. Anyway, things should be working as expected now. Fixes pushed to https://github.com/Hyperobjekt/spi-map/pull/53
